### PR TITLE
KBD19x: fixup the indicator support

### PR DESCRIPTION
**Pull request by mechmerlin**
## Description

The board has 3 indicators that weren't being used in the code. This commit fixes it. 

## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by zvecr — CHANGES_REQUESTED** [review]
Should use <https://docs.qmk.fm/#/feature_led_indicators?id=configuration-options> instead.

**Comment by mechmerlin** [comment]
> Should use https://docs.qmk.fm/#/feature_led_indicators?id=configuration-options instead.

yep that was the issue

**Comment by zvecr** [comment]
The default implementation is basically what you have added back.

File: keyboards/kbdfans/kbd19x/kbd19x.c [hunk 1]
```diff
@@ -16,3 +16,23 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
+}
+
+bool led_update_kb(led_t led_state) {
+    if (led_update_user(led_state)) {
+        writePin(B0, !led_state.num_lock);    // Left Indicator LED
+        writePin(B1, !led_state.caps_lock);   // Top Indicator LED
+        writePin(B2, !led_state.scroll_lock); // Right Indicator lED
+    }
+    return true;
+}
```
Comment (drashna) [inline keyboards/kbdfans/kbd19x/kbd19x.c:38]:
```suggetion
```

If you add to config.h...
```c
#define LED_NUM_LOCK_PIN B0
#define LED_CAPS_LOCK_PIN B1
#define LED_SCROLL_LOCK_PIN B2
```

https://docs.qmk.fm/#/feature_led_indicators?id=configuration-options

**Pull request closed without merging**